docs/environments/gke.md

```diff
@@ -63,3 +63,69 @@ kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=clu
 
 Once all pods are running you can return to the [User Guide](../userguide.md) to deploy the
 KubeFed control-plane.
+
+## Additional firewall rule for kubefed-admission-webhook in GKE private clusters
+
+Admission control in [GKE private clusters](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters)
+requires an additional VPC network firewall rule.
+
+### Issue
+
+The `kubefed-controller-manager` pod fails to start up when attempting to create
+a `KubeFedConfig` resource. A timeout error is observed.
+
+```
+Error updating KubeFedConfig "kube-federation-system/kubefed": Timeout: request did not complete within requested timeout 30s
+```
+
+### Background
+
+The GKE Kubernetes control plane is deployed in a Google-managed account.
+
+During the provisioning of a private cluster, GKE automatically creates:
+1. A network [peering connection](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#features)
+to allow the master to communicate to the node pools.
+1. A [firewall rule](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules)
+to allow ports 443 and 10250 from the master to the node pools.
+
+The GKE API Server operates with [`--enable-aggregator-routing=true`](https://github.com/kubernetes/kubernetes/issues/79739#issuecomment-509813068).
+This means that the Service IPs are internally translated to endpoint IPs and
+routed to the relevant pods.
+
+A number of [webhook configurations are deployed](../../charts/controllermanager/templates/webhook.yaml)
```

Comment:
Please verify the link works as it is missing an additional `charts/kubefed`.

Reply:
Sorry, I didn't test this after changing it. It should work now.